deployer/utils.py

```diff
@@ -130,6 +131,20 @@ def verify_and_decrypt_file(encrypted_path):
         return
 
 
+@contextmanager
+def get_decrypted_files(files, abspath):
+    """
+    This is a context manager that when entered provides a list of
+    file paths, where temporary files have been created if needed for
+    files that were encrypted and first need to be decrypted.
```

Comment:
```suggestion
    This is a context manager that when entered provides a list of
    file paths to temporary files that hold the decrypted version of
    encrypted files in `files`.
```
Question: if any file in the `files` list is not encrypted, what happens? Do we raise an error, or we just leave it as it is and add it to the list of decrypted files?

Reply:
https://github.com/2i2c-org/infrastructure/blob/cc32b3c71b532ee83d3a63939cbeb55afbd6e8af/deployer/utils.py#L89-L116

Reply:
@sgibson91, I believe we should modify the function description a bit. Otherwise we're describing what `verify_and_decrypt` does. 

```suggestion
    This is a context manager that combines multiple `verify_and_decrypt_file`
    context managers that open and/or decrypt the files in `files`.
```